.travis.yml

```diff
@@ -31,10 +31,14 @@ script:
       cd docs;
       make html;
       cd ..;
-      python -m doctr deploy --gh-pages-docs . --key-path deploy_key.enc;
-      python -m doctr deploy --gh-pages-docs docs --key-path deploy_key.enc;
-      python -m doctr deploy --no-require-master --gh-pages-docs "docs-$TRAVIS_BRANCH" --built-docs docs/_build/html --key-path deploy_key.enc;
-      python -m doctr deploy --no-require-master --key-path deploy_key.enc --no-sync --command "echo test";
+      if [[ -z "${DOCTR_DEPLOY_ENCRYPTION_KEY}" ]]; then
+          echo "DOCTR_DEPLOY_ENCRYPTION_KEY not set, likely building a PR from untrusted repository";
```

Comment:
"We ought" meaning we should. I don't know if we actually are (but if we aren't, we need to be). Someone actually reported this same error on SymPy, so I think there may be a real bug here.

In any case, let's fix this in a separate PR.